### PR TITLE
cygwin test fix:uv_resident_set_memory

### PR DESCRIPTION
**Pull request by damon-kwok**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -69,6 +69,9 @@ ipch
 # sphinx generated files
 /docs/build/
 
+# cmake generated files
+build/
```
Comment (bnoordhuis) [inline .gitignore]: This subsumes the `/build/gyp` rule on line 45. You can remove that one. Can you also add a leading slash?

Comment (damon-kwok) [reply .gitignore]: ok

**Comment by bnoordhuis** [comment]
https://ci.nodejs.org/job/libuv-test-commit/1108/

**Comment by cjihrig** [comment]
Landed in a24e8a17e5c72b1f81dc228197a8a414f6dd6fa0 and 2749d0c30da58161e24cfff9e17e530c4eb31fd2. Thanks!

**Pull request closed without merging**